RELEASE.md

```diff
@@ -265,6 +269,7 @@ mention the version explicitly. These files are listed below:
 #
 gradle.properties
 R/rdeephaven/DESCRIPTION
+web/client-api/types/package.json
```

Comment:
Does package-lock.json need to be updated too?

Reply:
I don't _think_ so, since I never had it in there previously, but I used the `npm version` command to perform this particular update and it emitted that. @mofojed can you weigh in?

Reply:
Yes, otherwise the next `npm install` will cause a `package-lock.json` diff (when it adds the version info to it). `package-lock.json` shouldn't typically be modified by hand though...
Side note - what version of `node`/`npm` are you using Colin? Doesn't seem to match the engines specified in the `package.json` (`"node": ">=16"`). We should update that version in https://github.com/deephaven/deephaven-core/blob/e09bcfdaca4264ecd06a73b0e4acab5f850e88ff/docker/registry/node/gradle.properties#L2 (probably use `node:18` now instead of 14).
We should have a [.nvmrc](https://github.com/deephaven/web-client-ui/blob/main/.nvmrc) file defining the node version of an [.npmrc](https://github.com/deephaven/web-client-ui/blob/main/.npmrc) specifying the engine version should be strict and enable provenance as well, so we can sign the builds using GitHub Actions: https://github.blog/2023-04-19-introducing-npm-package-provenance/
I guess then we should just have a gradle action that runs the `npm version`?

Reply:
I used npm 10.2.3 locally to run the `npm version` command that wrote that update to the package-lock.json, and must have created the file using the older node/npm from the container, since it produces a different package-lock.json output if i run it again outside that image.

If you're suggesting running npm version from CI, wouldn't that imply either creating a commit, or not updating the version itself?

Also, does provenance buy us anything here, given that it is just the .d.ts file?

Reply:
Bumping the node image breaks the raw-js-openapi build, i'll file an issue and get that worked out separately. I'll back out the `version` change to package-lock.json for the moment, and work out the node bump and other CI actions later?

Reply:
https://github.com/deephaven/deephaven-core/issues/4888 to follow up on the node image and updating these js projects.

Reply:
Looks like this fixes #4888 as well now? Looks good to me, but the instructions should be to run the a gradle bump action instead of manually editing this package.json file